docs/installation/command-line-tool.md

```diff
@@ -25,6 +33,16 @@ To install the bleeding edge version:
 BLEEDING_EDGE='true' bash <(curl https://raw.githubusercontent.com/konveyor/move2kube/main/scripts/install.sh)
 ```
 
+To install without sudo
+```
+USE_SUDO=false bash <(curl https://raw.githubusercontent.com/konveyor/move2kube/main/scripts/install.sh)
+```
+
+By default the script installs to `/usr/local/bin`. To install to a different directory
+```
+MOVE2KUBE_INSTALL_DIR=/my/new/install/dir bash <(curl https://raw.githubusercontent.com/konveyor/move2kube/main/scripts/install.sh)
+```
```

Comment:
We can combine these two into a single one, that's the most common use case. Also, we need to mention that the path needs to be added to PATH